examples/getting_started/run.py

```diff
@@ -41,19 +44,21 @@
     # take a few steps
     for i in range(NUM_STEPS):
         if config.SIMULATION_CONTROLLER.AGENT == "SphereAgent":
-            obs, reward, done, info = env.step(action={"apply_force": np.array([1.0, 1.0], dtype=np.float32)})
+            obs, reward, done, info = env.step(action={"add_force": np.array([10000.0, 0.0, 0.0], dtype=np.float64)})
             if not args.benchmark:
                 spear.log("SphereAgent: ")
-                spear.log(obs["compass"])
-                spear.log(obs["camera.final_color"].shape, obs["camera.final_color"].dtype)
+                spear.log("position:", obs["position"])
+                spear.log("rotation:", obs["rotation"])
+                spear.log("camera:", obs["camera.final_color"].shape, obs["camera.final_color"].dtype)
                 spear.log(reward, done, info)
-        elif config.SIMULATION_CONTROLLER.AGENT == "OpenBotAgent":
-            obs, reward, done, info = env.step(action={"apply_voltage": np.array([1.0, 0.715], dtype=np.float32)})
+        elif config.SIMULATION_CONTROLLER.AGENT == "WheeledVehicleAgent":
+            obs, reward, done, info = env.step(action={"apply_voltage": np.array([1.0, 1.0], dtype=np.float64)})
```

Comment:
action should be called `set_duty_cycle`

Reply:
Updated this